finality-aleph/src/nodes/mod.rs

```diff
@@ -30,11 +32,14 @@ pub mod testing {
 /// Max amount of tries we can not update a finalized block number before we will clear requests queue
 const MAX_ATTEMPTS: u32 = 5;
 
-struct JustificationParams<B: Block, H: ExHashT, C, BB> {
+struct JustificationParams<B: Block, H: ExHashT, C, BB>
+where
```

Comment:
If you are doing a `where` here already then maybe put the other bounds here as well.